packages/advanced-logic/src/advanced-logic.ts

```diff
@@ -63,10 +65,11 @@ export default class AdvancedLogic implements AdvancedLogicTypes.IAdvancedLogic
       erc777Stream: new Erc777Stream(),
       feeProxyContractEth: new FeeProxyContractEth(),
       anyToEthProxy: new AnyToEthProxy(currencyManager),
-      nativeToken: [new NearNative()],
-      anyToNativeToken: [new AnyToNear(currencyManager)],
+      nativeToken: [new NearNative(), new NearTestnetNative()],
+      anyToNativeToken: [new AnyToNear(currencyManager), new AnyToNearTestnet(currencyManager)],
```

Comment:
If we go down that line, can we add `anyToEthProxy` and `feeProxyContractEth` to these 2 generic PNs? And deprecate the "separate" use of these extensions? It does not add anything while we just have Near.

Reply:
I agree! We should deprecate `any-to-eth-proxy` and instead use `any-to-native-token`. Not sure however that `eth-fee-proxy-contract` and `native-token` are compatible as the latest doesn't support fees as far as I know. I'll keep this PR a refactor without changing payment networks, but let's keep this in mind for the next steps.